src/main/java/org/dinosaur/foodbowl/domain/store/dto/StoreRequest.java

```diff
@@ -0,0 +1,57 @@
+package org.dinosaur.foodbowl.domain.store.dto;
+
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import java.math.BigDecimal;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class StoreRequest {
+
+    @NotBlank(message = "가게 이름은 반드시 입력되어야 합니다.")
+    private String storeName;
+
+    @NotBlank(message = "주소는 반드시 입력되어야 합니다.")
+    private String addressName;
+
+    @NotBlank(message = "광역시 또는 도는 반드시 입력되어야 합니다.")
+    private String region1depthName;
+
+    @NotBlank(message = "시/군/구는 반드시 입력되어야 합니다.")
+    private String region2depthName;
+
+    @NotBlank(message = "읍/면/동은 반드시 입력되어야 합니다.")
+    private String region3depthName;
+
+    private String roadName;
+
+    @Pattern(regexp = "^[YN]$")
+    private String undergroundYN;
+
+    private String mainBuildingNo;
+
+    private String subBuildingNo;
+
+    private String buildingName;
+
+    private String zoneNo;
+
+    @NotNull(message = "경도는 반드시 입력되어야 합니다.")
+    @DecimalMin(value = "-180", message = "경도의 최소값은 -180입니다.")
+    @DecimalMax(value = "180", message = "경도의 최대값은 180입니다.")
```

Comment:
`@DecimalMin`, `@DecimalMax`에 대해 배워갑니다 🥳

```suggestion
    @NotNull(message = "경도는 반드시 입력되어야 합니다.")
    @DecimalMin(value = "-180", message = "경도의 최소값은 {value}입니다.")
    @DecimalMax(value = "180", message = "경도의 최대값은 {value}입니다.")
```
다음과 같이 하면 앞의 value값을 가져와서 사용할 수 있습니다!